commands/serverless-extra.go

```diff
@@ -155,19 +155,23 @@ func RunServerlessExtraGetMetadata(c *CmdConfig) error {
 	if err != nil {
 		return err
 	}
```

Comment:
Changes in this file are to adjust to the signature change in `ReadProject`.   It returns a `ProjectSpec` rather than ready-for-output information.